### DISCUSSION TITLE
This Deployment has been disabled.  Your connection is working correctly.  Vercel is working correctly

### DISCUSSION THREAD

**Discussion by abidraza5594** [vercel/community#2811] [Help] [answered]
### Page to Investigate

This Deployment has been disabled.  Your connection is working correctly.  Vercel is working correctly

### Steps to Reproduce

how to fix this error 
This Deployment has been disabled.

Your connection is working correctly.

Vercel is working correctly

**Comment by swarnava** [comment]
Hi,

What is the deployment url?

**Comment by abidraza5594** [comment]
https://meesho-clone-sooty.vercel.app/

This is a deployed link

**Comment by swarnava** [comment, marked as answer]
This was disabled for violating our [Fair Usage Policy](https://vercel.com/docs/concepts/limits/fair-use-policy) and [Terms of Service](https://vercel.com/legal/terms).
